src/plugins/vis_builder/public/application/components/workspace.tsx

```diff
@@ -44,15 +46,17 @@ export const Workspace: FC = ({ children }) => {
     async function loadExpression() {
       const schemas = ui.containerConfig.data.schemas;
 
-      const noAggs = aggConfigs?.aggs?.length === 0;
+      const noAggs = (aggConfigs?.aggs?.length ?? 0) === 0;
```

Comment:
opinion - this reads worse to me than

`      const noAggs = !aggConfigs?.aggs?.length;`
or even
`      const noAggs = aggConfigs?.aggs?.length == false;`

Also, for conditionals like this I'd prefer `hasNoAggs`.